proto/invocation.proto

```diff
@@ -524,4 +529,6 @@ message GetTrendResponse {
 
   // The list of execution stats found.
   repeated ExecutionStat execution_stat = 3;
+
+  bool has_invocation_stat_percentiles = 4;
```

Comment:
Discussed offline. invocation_stat_percentiles depend on the olap flag on the backend. so it would be tricky if we have a FE flag. For example, when the FE flag is on and olap db is off for example.